microsoft-edge/progressive-web-apps-chromium/how-to/index.md

```diff
@@ -124,11 +132,13 @@ Service workers are specialized [Web Workers](https://developer.mozilla.org/docs
 
 Service workers are defined in a special JavaScript file, described in [Using Service Workers](https://developer.mozilla.org/docs/Web/API/Service_Worker_API/Using_Service_Workers) and [Service Worker API](https://developer.mozilla.org/docs/Web/API/Service_Worker_API).
 
-To build a service worker in your project, use the **Cache-first network** service worker recipe from PWA Builder, as follows.
+To build a service worker in your project, use the **Cache-first network** service-worker recipe from PWA Builder, as follows.
```

Comment:
"this phrase" could mean "service worker recipe" or just "service worker".  The phrase "service worker recipe", case-sensitive, only occurs one time in the repo: this instance.  The only other variant is: "Service Worker recipes", which shows an alternative solution instead of hyphen: init caps.  I might change this instance to init caps instead of hyphen.

See MWSG https://docs.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/hyphens#noun-modifiers